void myFunction(String test) {
    try{
        makeMyCodeGreatAgain();
    }
    throw new Exception("Test");
}